scripts/update-ca-bundle.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2008. All rights reserved.
+#
+
+set -e
+
+mkdir -p /etc/stunnel
+find /etc/stunnel/certs -name '*.pem' | xargs cat > /etc/stunnel/xapi-stunnel-ca-bundle.pem
```

Comment:
Would be better if this created a temporary file during concatenation and atomically moved it when complete to avoid the situation where `stunnel` could see an incomplete file.

Also how is stunnel going to notice that there is a new certificate, do you need to signal it, restart it somehow?

Reply:
Thanks for your comments! @edwintorok 
Just to confirm, do you mean it's better to create a temp file to put all the content inside of `*.pem`? After that `cat` the content into `xapi-stunnel-ca-bundle.pem` to avoid incomplete file.
Like:
```find /etc/stunnel/certs -name '*.pem' | xargs cat > /etc/stunnel/temp 
cp /etc/stunnel/temp  /etc/stunnel/xapi-stunnel-ca-bundle.pem
rm -f /etc/stunnel/temp```

For the second, if there is a new certificate, the user should install the certificate with `xe pool-certificate-install`

Reply:
`cp` is not atomic and would have the same problem as original `cat`, please use `mv /etc/stunnel/xapi-stunnel-ca-bundle.pem.tmp /etc/stunnel/xapi-stunnel-ca-bundle.pem`

Reply:
Updated, thanks!